.github/workflows/ci.yaml

```diff
@@ -52,19 +56,23 @@ jobs:
     permissions:
       id-token: write # This is required for requesting the JWT
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+      - name: Set up Python 
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           pip install .
-      # See https://github.com/aws-actions/configure-aws-credentials for configuring
+      # See https://github.com/Moulick/configure-multiple-aws-roles for configuring
       # the aws side of this - the below action is just a light wrapper
       - name: Configure AWS Credentials
-        uses: mcblair/configure-aws-profile-action@v0.1.1
+        uses: Moulick/configure-multiple-aws-roles@v4
```

Comment:
So, dusting this off - the original action is using an AWS API which is about to be deprecated. I've been watching [this PR](https://github.com/mcblair/configure-aws-profile-action/pull/1#event-11695267238) waiting for a fix to go in. This ended up being resolved via a fork.